src/java.compiler/share/classes/javax/lang/model/element/VariableElement.java

```diff
@@ -93,4 +98,18 @@ public interface VariableElement extends Element {
      */
     @Override
     Element getEnclosingElement();
+
+    /**
+     * {@return {@code true} if this is an unnamed variable and {@code
+     * false} otherwise}
+     *
+     * @implSpec
+     * The default implementation of this method returns {@code
+     * false}.
+     *
+     * @jls 6.1 Declarations
+     * @jls 14.4 Local Variable Declarations
+     */
+    @PreviewFeature(feature=PreviewFeature.Feature.UNNAMED, reflective = true)
+    default boolean isUnnamed() { return false; }
```

Comment:
Looking at the various sources more closely, I'll revise my previous guidance and recommend the default method be defined as:
return getSimpleName().isEmpty()
which would also avoid the strict need to override the method in Symbol.